src/backend/distributed/executor/multi_executor.c

```diff
@@ -690,3 +698,16 @@ AlterTableConstraintCheck(QueryDesc *queryDesc)
 
 	return true;
 }
+
+
+/*
+ * ExecutorBoundParams returns the bound parameters of the current ongoing call
+ * to ExecutorRun. This is meant to be used by UDFs which need to access bound
+ * parameters.
```

Comment:
what happens when a PL/pgSQL UDF calls other PL/pgSQL UDFs? We'd be in multiple levels of ExecutorRun. Could we lose the executorBoundParams?

Reply:
Fixed & added a test which would fail if this is not handled properly.